packages/web-components/src/horizontalbarchart/horizontalbarchart.ts

```diff
@@ -293,15 +296,28 @@ export class HorizontalBarChart extends FASTElement {
 
     let tooltip: any;
 
-    containerDiv
+    const chartTitleDiv = containerDiv.append('div').attr('class', 'chartTitleDiv');
+    chartTitleDiv
+      .append('div')
       .append('span')
       .attr('class', 'chartTitle')
       .text(data?.chartTitle ? data?.chartTitle : '');
 
+    const hideNumber = this.hideRatio === undefined ? false : this.hideRatio;
+
+    const showRatio = this.variant !== Variant.AbsoluteScale && !hideNumber && data!.chartData!.length === 2;
+    const getChartData = () => (data!.chartData![0].data ? data!.chartData![0].data : 0);
+
+    if (showRatio) {
+      const ratioDiv = chartTitleDiv.append('div').attr('role', 'text');
+      ratioDiv.append('span').attr('class', 'ratioNumerator').text('40');
```

Comment:
why hardcoding these values

Reply:
Was hardcoded in v8 chart also. Should change this ?

Reply:
Changed.

Reply:
can you show where it was hardcoded.

Reply:
My bad. I remembered something else